Update synthetic-test-monitors.md (#22003)
DataDog/documentation: content/en/synthetics/guide/synthetic-test-monitors.md

## Create a Synthetic test monitor

> <div class="alert alert-info">You cannot create or import a Synthetic test monitor in <a href="/monitors/">Monitors</a>. If you'd like to define additional monitors on these tests, look at <a href="synthetics/metrics/">Synthetic Monitoring & Continuous Testing Metrics</a>. </div>

Reviewer: ```suggestion
<div class="alert alert-info">You cannot create or import a Synthetic test monitor in <a href="/monitors/">Monitors</a>. To define additional monitors on Synthetic tests, see <a href="/synthetics/metrics/">Synthetic Monitoring & Continuous Testing Metrics</a>. </div>
```
